plugins/autosuspend.koplugin/main.lua

```diff
@@ -1,12 +1,7 @@
 local Device = require("device")
 
-if not Device:isCervantes() and
-    not Device:isKindle() and
-    not Device:isKobo() and
-    not Device:isRemarkable() and
-    not Device:isSDL() and
-    not Device:isSonyPRSTUX() and
-    not Device:isPocketBook() then
+-- Any device that can Suspend can also at least power off  ;).
```

Comment:
Hold on, what does that mean for the emulator? :-P

Reply:
Okay, yeah, it's a lie for the emu ;p.

Reply:
I mean, it could just close the program or something. :-P

Reply:
Oh, actually, I thought the Emulator was `!canPoweroff`, but that's not the case, so, err, good question, I'll check what it actually tries to do ;p.

Reply:
Okay, yeah, the actual methods are not implemented, so we just exit, which works out, I guess ;).